cmake/Modules/FindTBB_MALLOC.cmake

```diff
@@ -6,7 +6,9 @@
 #  TBB_MALLOC_FOUND        - True if tbb found.
 #
 
-
```

Comment:
I had to look up what this policy is, so a comment why this is needed would be nice.